lib/catalog/martyrology.ts

```diff
@@ -13,6 +13,18 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Aaron',
       titles: [Title.Martyr],
     },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureux-achard-eveque-davranches
+    // - https://en.wikipedia.org/wiki/Achard_of_Saint_Victor
+    achard_of_saint_victor_bishop: {
+      canonizationLevel: CanonizationLevels.Blessed,
+      name: 'Achard',
+      titles: [Title.Bishop],
+      dateOfBirth: 1100,
+      dateOfBirthIsApproximative: true,
+      dateOfDeath: 1171,
```

Comment:
Do we want to add the full date of his death? :thinking: Based on [Wikipedia](https://en.wikipedia.org/wiki/Achard_of_Saint_Victor), he died on 29 March 1171.

```suggestion
      dateOfDeath: '1171-03-29',
```